lib/DidKeyDriver.js

```diff
@@ -3,29 +3,54 @@
  */
 import * as didIo from '@digitalbazaar/did-io';
 import {
-  getDid, getKey, getKeyAgreementKeyPair, setVerificationKeyPairId
+  getDid, getKey, getKeyAgreementKeyPair, getMultiKeyHeader,
+  setVerificationKeyPairId
 } from './helpers.js';
 import {
   Ed25519VerificationKey2020
 } from '@digitalbazaar/ed25519-verification-key-2020';
 
+const Ed25519Header = 'z6Mk';
+
 export class DidKeyDriver {
-  /**
-   * @param {object} options - Options hashmap.
-   * @param {object} [options.verificationSuite=Ed25519VerificationKey2020] -
-   *   Key suite for the signature verification key suite to use.
-   */
-  constructor({verificationSuite = Ed25519VerificationKey2020} = {}) {
+  constructor() {
     // used by did-io to register drivers
     this.method = 'key';
-    this.verificationSuite = verificationSuite;
+    this.allowedKeyTypeHandlers = new Map();
+    // Set default keyTypeHandler for Ed25519  multikey header type.
+    this.allowedKeyTypeHandlers.set(
+      Ed25519Header, Ed25519VerificationKey2020
+    );
   }
 
+  /**
+   * Registers an allowed keytype handler for a multikey header.
+   *
+   * @param {object} options - Options hashmap.
+   *
+   * @param {string|Array<string>} options.multikeyHeaders - The multikey
+   *   headers to register.
+   * @param {Function} options.keyTypeHandler - The `keyTypeHandler` to map
+   *   the headers to.
+   */
+  registerKeyTypeHandler({multikeyHeaders, keyTypeHandler}) {
```

Comment:
While this is more verbose, we should do it -- in case we handle to additively support using a multikey header later (which would be a binary value, not a string).

I think we can probably shorten `registerKeyTypeHandler` to just `use` though -- I don't think we expect this driver to use anything other than handlers for key types.

```suggestion
  use({multibaseMultikeyHeader, keyTypeHandler}) {
```

Reply:
Updated [7225dc3](https://github.com/digitalbazaar/did-method-key/pull/55/commits/7225dc397c6707a8c4b7af67e8a70b32a96c731b)